holoviews/core/spaces.py

```diff
@@ -853,6 +853,11 @@ def unbounded(self):
                 unbounded_dims.append(str(kdim))
         return unbounded_dims
 
+    @property
+    def current_key(self):
+        """Returns the current key value."""
+        return getattr(self, '_current_key', None)
```

Comment:
I think I would prefer you declare `self._current_key=None` in the constructor and just return `self._current_key`. That way you can prevent anyone from overwriting `current_key` without needing to use `getattr` here.

Reply:
Done!